### PR TITLE
Tutorial Advection_F generalized to work in both 2d and 3d

### PR DESCRIPTION
**Pull request by nataraj2**
make DIM=2 (or DIM=3) will compile the code. The 3d fortran routines from Advection_AmrCore are reused.

### CODE REVIEW

**Comment by atmyers** [comment]
This code does not compile for me if I do `make` in `Tutorials/Amr/Advection_F/Exec/SingleVortex`.

**Comment by nataraj2** [comment]
Sorry about that. Have fixed it. Will take care in future.

File: Tutorials/Amr/Advection_F/Exec/SingleVortex/Make.package [hunk 1]
```diff
@@ -1,3 +1 @@
-f90EXE_sources += Prob.f90
-
-F90EXE_sources += face_velocity_$(DIM)d.F90
+f90EXE_sources += Prob_$(DIM)d.f90 face_velocity_$(DIM)d.f90
```
Comment (atmyers) [inline Tutorials/Amr/Advection_F/Exec/SingleVortex/Make.package]: This compiles for me in 3D now, but not 2D. I think the problem is a capitalization issue - it's looking for `face_velocity_2d.f90` but the file is called `face_velocity_2d.F90`. Are you testing this on a Mac? You might miss this if your file system is not case-sensitive.

Comment (nataraj2) [reply Tutorials/Amr/Advection_F/Exec/SingleVortex/Make.package]: Mac - yes. Even though I had face_velocity_2d.f90 (small) on my system the git did not change it. It remained capital .F90 in the repo. Had to remove and add again. Have fixed it now. Sorry for the hassle.

File: Tutorials/Amr/Advection_F/Exec/SingleVortex/inputs.rt [hunk 2]
```diff
@@ -1,31 +0,0 @@
-max_step  = 1000000
```
Comment (atmyers) [inline Tutorials/Amr/Advection_F/Exec/SingleVortex/inputs.rt]: Why have these files been removed? `inputs.rt` is needed by our regression tests.

File: Tutorials/Amr/Advection_F/Exec/SingleVortex/inputs.tracers [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# ------------------  INPUTS TO MAIN PROGRAM  -------------------
```
Comment (atmyers) [inline Tutorials/Amr/Advection_F/Exec/SingleVortex/inputs.tracers]: Similarly, this inputs file is not needed by this problem - it pertains to tracers, but tracer particles are not currently available in the Fortran interfaces.

Comment (nataraj2) [reply Tutorials/Amr/Advection_F/Exec/SingleVortex/inputs.tracers]: You can close this one. Opening a new one with newly created fork.

**Pull request closed without merging**